### PR TITLE
docs/managed-victoriametrics: update documentation

### PR DESCRIPTION
**Pull request by dmitryk-dk**
Updated documentation for managed VictoriaMetrics

Signed-off-by: dmitryk-dk [d.kozlov@victoriametrics.com](mailto:d.kozlov@victoriametrics.com)

### CODE REVIEW

**Comment by tenmozes** [comment]
please hide all tokens or leave normal but pleas choose one approach

File: docs/managed-victoriametrics/overview.md [hunk 1]
```diff
@@ -22,15 +22,14 @@ The most common use cases for VictoriaMetrics are:
 * Replacement for InfluxDB and OpenTSDB, which uses lower amounts of RAM, CPU and disk;
 * Cost-efficient alternative for DataDog.
 
-We are glad to announce the availability of Managed VictoriaMetrics 
-at AWS Marketplace - [try it right now](https://aws.amazon.com/marketplace/pp/prodview-4tbfq5icmbmyc)!
+We are glad to announce general availability of Managed VictoriaMetrics – [try it right now](https://cloud.victoriametrics.com/signUp?utm_source=website&utm_campaign=docs_overview) and get $200 of credits!
```
Comment (tenmozes) [inline docs/managed-victoriametrics/overview.md:25]: let's remove that we glad to announce we did that :)

File: docs/managed-victoriametrics/overview.md [hunk 2]
```diff
@@ -39,4 +38,4 @@ Managed VictoriaMetrics comes with the following features:
 * Every Managed VictoriaMetrics instance runs in an isolated environment, so instances cannot interfere with each other;
 * Managed VictoriaMetrics instance can be scaled up or scaled down in a few clicks;
 * Automated backups;
-* Pay only for the actually used compute resources - instance type, disk and network.
+* Pay only for the actually used resources - instance type, disk and network.
```
Comment (tenmozes) [inline docs/managed-victoriametrics/overview.md:41]: should we use the same measurements as in billing?
compute, storage, traffic?

File: docs/managed-victoriametrics/quickstart.md [hunk 3]
```diff
@@ -11,179 +11,265 @@ aliases:
...
+   <p>
+     <img src="how_to_add_payment_method_invalid_card.png" width="800">
+   </p>
+1. Successfully added card will be shown on the page as follows:
+   <p>
+     <img src="how_to_add_payment_method_card_added.png" width="800">
+   </p>
 
-If you forgot password, it can be restored in the following way:
+### Pay via AWS Marketplace
```
Comment (tenmozes) [inline docs/managed-victoriametrics/quickstart.md:107]: Link your AWS billing account

File: docs/managed-victoriametrics/quickstart.md [hunk 4]
```diff
@@ -11,179 +11,265 @@ aliases:
...
+</p>
+
+If the payment method was changed successfully, the following message will appear: 
+
+<p>
+  <img src="change_payment_method_success.png" width="800">
+</p>
 
-1. Enter your email in the field and click `Send Email` button:
+## Restoring a password
```
Comment (tenmozes) [inline docs/managed-victoriametrics/quickstart.md:156]: Password restoration?

File: docs/managed-victoriametrics/quickstart.md [hunk 5]
```diff
@@ -11,179 +11,265 @@ aliases:
...
-Follow usage example in order to configure access to VictoriaMetrics for your Prometheus, 
+Follow usage examples in order to configure access to VictoriaMetrics for your Prometheus, 
 Grafana or any other software.
 
-## Modifying deployment
+## Modifying an existing deployment
 
-Remember, you always can add, remove or modify existing deployment by changing their size or any parameters on the 
-update form.
+Remember, you can always add, remove or modify existing deployment by changing its size or any parameters on the
```
Comment (tenmozes) [inline docs/managed-victoriametrics/quickstart.md:256]: please mention that downgrade for cluster is currently not available (if this is a right place for that)

File: docs/managed-victoriametrics/quickstart.md [hunk 6]
```diff
@@ -11,179 +11,265 @@ aliases:
...
 
 <p>
-  <img src="quickstart_customise-deployment.png" width="800">
+  <img src="modify_deployment_additional_settings.png" width="800">
 </p>
 
-In that section additional params can be set:
+In that section, additional params can be set:
 
 * `Deduplication` defines interval when deployment leaves a single raw sample with the biggest timestamp per each discrete interval;
```
Comment (tenmozes) [inline docs/managed-victoriametrics/quickstart.md:271]: please add link to VM docs for deduplication

File: docs/managed-victoriametrics/quickstart.md [hunk 7]
```diff
@@ -11,179 +11,265 @@ aliases:
...
-* `Maintenance Window` when deployment should start upgrade process if needed;
+* `Maintenance Window` when deployment should start an upgrade process if needed;
 * `Settings` allow to define different flags for the deployment.
```
Comment (tenmozes) [inline docs/managed-victoriametrics/quickstart.md:273]: please provide link to cluster and signle flags

File: docs/managed-victoriametrics/user-managment.md [hunk 8]
```diff
@@ -141,84 +137,85 @@ In the table, there is additional information about the users:
...
 You will be redirected to the main page with a success or error message
 
 <p>
-    <img src="user-managment_delete-user-success.png" width="800">
+    <img src="user_management_delete_sucess.png" width="800">
+</p>
+
+## How to resend invitation
+
+If invitation was expired it is possible to resend email to the user
```
Comment (tenmozes) [inline docs/managed-victoriametrics/user-managment.md:203]: is expired

File: docs/managed-victoriametrics/overview.md (hunk 2 above)
Comment (dmitryk-dk) [reply docs/managed-victoriametrics/overview.md]: I used the same measurements. Users will better understand our information in the site

File: docs/managed-victoriametrics/quickstart.md (hunk 3 above)
Comment (dmitryk-dk) [reply docs/managed-victoriametrics/quickstart.md]: Updated

File: docs/managed-victoriametrics/quickstart.md (hunk 4 above)
Comment (dmitryk-dk) [reply docs/managed-victoriametrics/quickstart.md]: Updated

File: docs/managed-victoriametrics/quickstart.md (hunk 5 above)
Comment (dmitryk-dk) [reply docs/managed-victoriametrics/quickstart.md:256]: added this warning in the text below

File: docs/managed-victoriametrics/quickstart.md (hunk 6 above)
Comment (dmitryk-dk) [reply docs/managed-victoriametrics/quickstart.md]: link was added

File: docs/managed-victoriametrics/quickstart.md (hunk 7 above)
Comment (dmitryk-dk) [reply docs/managed-victoriametrics/quickstart.md]: links for cluster components flags and single version was added

File: docs/managed-victoriametrics/user-managment.md (hunk 8 above)
Comment (dmitryk-dk) [reply docs/managed-victoriametrics/user-managment.md]: fixed

**Comment by dmitryk-dk** [comment]
> please hide all tokens or leave normal but pleas choose one approach

fixed. Hidden all tokens to be consistent with other parts of the documentation

**Review by tenmozes — APPROVED** [review]

**Review by hagen1778 — APPROVED** [review]

**Comment by valyala** [comment]
FYI, the commit created from this pull request - https://github.com/VictoriaMetrics/VictoriaMetrics/commit/a6d06474f468e270e61f3327392983b6d6f96dba  - takes 25MB! Such big commits negatively impact VictoriaMetrics repository size. It would be great to reduce the size of images stored in our docs next time. The same applies to all the images used in any VictoriaMetrics properties. There is little sense in using 1MB images instead of 10Kb images for most web properties and docs. @dmitryk-dk , @yatskevich .

**Comment by hagen1778** [comment]
@valyala the attempt to reduce the size of images is made here https://github.com/VictoriaMetrics/VictoriaMetrics/pull/5206

**Pull request merged**